cbor/event.go

```diff
@@ -83,6 +84,7 @@ func CreateEvent(
 			Node: codedHeader,
 			obj:  eventHeader,
 		},
+		body: codedBody,
```

Comment:
Not sure why this was never included in the object returned from `CreateEvent`. Should lead to less trips to the `DAGService`.

Reply:
Nice